db/migrate/20170504145330_sync_whitehall_public_updated_at.rb

```diff
@@ -0,0 +1,29 @@
+class SyncWhitehallPublicUpdatedAt < ActiveRecord::Migration[5.0]
+  def change
+    schema_names = %w(gone redirect special_route speech statistical_data_set take_part topical_event_about_page)
+
+    schema_names.each do |schema_name|
+      scope = Edition.where(publishing_app: "whitehall")
+                     .where(schema_name: schema_name)
+                     .where("first_published_at AT TIME ZONE 'Europe/London' > public_updated_at")
```

Comment:
This is defensive thinking on my part, the `first_published_at` can be set by the admin user (eg. speeches) which may contain a value with an implicit BST offset.
The record count differs with an without this, the scope is narrowed slightly by doing the query with the timezone, which might mean some genuine < 1 hour discrepancies still exist. I'd expect to have another pass at this given the volume of data.